pallets/gear/src/lib.rs

```diff
@@ -497,9 +498,26 @@ pub mod pallet {
                 // Update message gas limit for it may have changed in the meantime
 
                 let msg_id = *dispatch.message_id();
+                let (gas_limit, _) = if let Some(limit) = T::GasHandler::get_limit(msg_id) {
+                    limit
+                } else {
+                    log::debug!(
+                        target: "essential",
+                        "No gas handler for message: {:?} to {:?}",
+                        dispatch.message_id(),
+                        dispatch.message.dest
+                    );
 
-                let (gas_limit, _) = T::GasHandler::get_limit(msg_id)
-                    .expect("Should never fail if ValueNode works properly");
+                    common::queue_dispatch(dispatch);
+
+                    let consumed = T::DbWeight::get().reads(1) + T::DbWeight::get().writes(1);
+                    Self::decrease_gas_allowance(consumed);
+                    if Self::gas_allowance() < consumed {
+                        break;
+                    }
```

Comment:
What's it? I think some comments wouldn't be useless

Reply:
Ok